### ISSUE TITLE
Ignore attribute is being ignored

### ISSUE THREAD

**Issue by redent** [oysteinkrog/SQLite.Net-PCL#66]
This simple example reproduces the issue:

```
public class DummyClass
{
    [PrimaryKey, AutoIncrement]
    public int Id { get; set; }

    public string Foo { get; set; }
    public string Bar { get; set; }

    [Ignore]
    public List<string> Ignored { get; set; }
}
```

This sample fails with an error on the `CreateTable` statement:

```
System.NotSupportedException : Don't know about System.Collections.Generic.List`1[System.String]
```

It shouldn't process the `Ignored` property at all.

**Comment by murali0009** [comment]
Any update for this?

**Comment by redent** [comment]
@murali0009 It was a bug introduced in 2.4.0 and fixed in 2.4.1. What kind of update do you need?

**Comment by murali0009** [comment]
@redent  Thanks for your reply. It was an oversight on my part.
